src/Microsoft.VisualBasic/ref/Configurations.props

```diff
@@ -2,7 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard-Unix;
```

Comment:
Reference assemblies cannot differ by runtime.  Runtime is never considered when resolving files that get passed to the compiler.